client/packages/common/src/intl/locales/fr/common.json

```diff
@@ -1153,6 +1158,7 @@
   "label.to-datetime": "jusqu'à la date/heure",
   "label.to-expiry": "Jusqu'à péremption",
   "label.to-shipped-date": "Jusqu'à la date d'expédition",
+  "label.to-shipped-datetime": "jusqu'à la date d'expedition",
```

Comment:
Are the rest of these just been merged in from somewhere? If not, is it correct that this is lower case?

Reply:
was just a sort 👀  but good point will caps

Reply:
Oh hm, maybe check the english? Maybe its used within another sentence or smth?

Reply:
Ah removed the shipped-date wasn't being used anywhere